src/base/shapes/ShapeStorage.js

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ *  Copyright 2017 Adobe, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+import * as THREE from 'three';
+import GlobalStyles from '../../globalStyles';
+import ShapesFactory from '../ShapesFactory';
+import ShapesUtils from '../ShapesUtils';
+
+class ShapeStorage {
```

Comment:
If you are adding shapes to this repo, it would be nice to have some screenshots in the PR.

Reply:
Will fix the colors in globalStyles and will attach screenshots for the shapes